public/app/features/plugins/plugin_loader.ts

```diff
@@ -183,11 +182,12 @@ export async function importAppPlugin(meta: PluginMeta): Promise<AppPlugin> {
     return importedAppPlugins[pluginId];
   }
 
+  throwIfAngular(meta);
```

Comment:
Super-nit: should we move this to the top of the function body? (Not like it could return any already imported app plugins from the importedAppPlugins "cache" otherwise.)